bigchaindb_driver/transport.py

```diff
@@ -1,50 +1,50 @@
+from time import time
+
+from requests.exceptions import ConnectionError
+
 from .connection import Connection
+from .exceptions import TimeoutError
 from .pool import Pool
-from .exceptions import TimeoutException, TransportError
-from datetime import datetime, timedelta
-from time import time
 
 
 class Transport:
-    """Transport class."""
+    """Transport class.
+
+    """
 
-    def __init__(self, *nodes, timeout=None, headers=None):
+    def __init__(self, *nodes, headers=None, timeout=None):
```

Comment:
are we using the `headers` here somewhere?

Reply:
Great catch! We do not need them here.